### ISSUE TITLE
Ability to close Lightbox with Mouse-Click

### ISSUE THREAD

**Issue by matthiasballreich** [arnowelzel/lightbox-photoswipe#75]
Would be cool to have an option to allow the PhotoSwipe Lightbox to close with a Mouse-Click on the image and not only on the background.

**Comment by arnowelzel** [comment]
Mouse clicks inside the image are already used to zoom.

**Comment by matthiasballreich** [comment]
Yes i saw this on my test environment. But maybe there coule be an option inetgrated to replace this behaviour, so that the users could select which action should be executed on mouse click insid the image?
